lore/species/tajaran/The Great Frost.md

```diff
@@ -0,0 +1,85 @@
+
+The next crisis to tackle became the lack of raw materials that were always in demand, lumber and iron were of great importance but no amount of scavenging could ever meet what was needed. As a means to an end they began to develop primitive yet sturdy machines in some holds. Robots of pure programming and no mind capable of operating longer than any man and in places they themselves could not go. These early machines were clunky but effective. Before the great frost the tajaran people were only just reaching the stage of true advancement in robotics. The ever praised Akari had equipped many holds with that they needed to continue this work, though few managed to create any more than two or three robots. Small in number or not, these machines were capable of working day and night performing simple tasks such as digging, mining, and the culling of trees. Over time with the revival of faith the position on what these tools meant varied among many, but some saw these machines as a gift from S�randarr and decorated them with religious symbols and carvings. In time, they were collectively called the frost breakers as their number and use swelled.
+
+Finally, as food and material scarcity no longer became the primary fear of the people attention now turned to the dangers of the frost land. The internal threat of the Shards of Messa and the external threats of dangerous predators became the primary concern. After the Great Frost began the vicious and durable predators from the poles began to migrate, spared in the first few decades due to distance the tajaran began to see increased presence of the deadliest predators in the wastelands. The most common of which was the Savik, vicious and brutal predators with a temper and viciousness to match their strength. Rarer among them were the White Renders, sometimes referred to as Messa�s Claws, huge hulking beasts long of limb and able to outrun even the fastest tajaran. Battles between these apex predators and the Sun Wardens became the most common cause of death. Holds without well constructed defenses also fell prey to attacks from these beasts searching for an easy meal. To combat this problem a new weapon was created, one which origins are unknown, yet many attribute its design to Akari Tal�Wrack himself.

```

Comment:
"... Savik, brutal predators with a temper..." Vicious is used twice here. 
"To combat this problem, a new weapon was created, one **whose** origins are unknown, yet..."